contrib/inventory/vagrant.py

```diff
@@ -56,19 +66,13 @@
 # helper functions
 #
 
+
 # get all the ssh configs for all boxes in an array of dictionaries.
 def get_ssh_config():
-    configs = []
-
-    boxes = list_running_boxes()
+    return {k: get_a_ssh_config(k) for k in list_running_boxes()}
 
-    for box in boxes:
-        config = get_a_ssh_config(box)
-        configs.append(config)
 
-    return configs
-
-#list all the running boxes
+# list all the running boxes
 def list_running_boxes():
     output = subprocess.check_output(["vagrant", "status"]).split('\n')
```

Comment:
@bcoca  `subprocess.check_output` was introduced in 2.7 https://docs.python.org/2/library/subprocess.html#subprocess.check_output. it was not compatible with 2.6 from the begining.